### ISSUE TITLE
Collapse openssl build/test jobs into one

### ISSUE THREAD

**Issue by rsbeckerca** [ituglib/ituglib-build#15]
The performance of OSSNS does not give advantage to running builds in parallel. This should be folded into a single job.

Cloned from #12

**Comment by rsbeckerca** [comment]
There is still work to do to consolidate all version builds into one, but that depends on deprecating 1.0.2 and 1.1. So this is basically on hold.

**Comment by rsbeckerca** [comment]
3.0.x and 1.1.1 series have separate jobs because they have separate repositories.
